### DISCUSSION TITLE
Input pins for otp verifications

### DISCUSSION THREAD

**Discussion by nayeshdaggula** [mantinedev/mantine#2622] [Feature requests]
Hi

Its very thank you for grate ui module.

if you guys can work on to add Input pins for otp verifications. its good have that option

Thank you

**Comment by zhulien-ivanov** [comment]
Checkout: https://github.com/mantinedev/mantine/tree/master/src/mantine-labs. Installation steps are included in the repo. The component you're looking for is `PinInput`.
